backend/data/helpers/recipeScraper.js

```diff
@@ -1,39 +1,109 @@
 
 const rp = require('request-promise');
 const $ = require('cheerio');
+const numericQuantity = require("numeric-quantity");
 
-checkUrl = (newRecipeObj) =>{
-  console.log(newRecipeObj);
-  const checkRecipeUrl = newRecipeObj.recipeUrl;
-  if(checkRecipeUrl.indexOf('allrecipes.com')){
-      allRecipeScraper(checkRecipeUrl);
-  }else if(checkRecipeUrl.indexOf('pinchofyum.com')){
-      pinchOfYumScraper(checkRecipeUrl);
+checkUrl = async (newRecipeObj) => {
```

Comment:
I don't think that you have to turn these into a promise. Are you seeing some errors coming up when you don't have async/await here?